java-checks/src/main/java/org/sonar/java/checks/spring/SpelExpressionCheck.java

```diff
@@ -55,7 +55,7 @@ public class SpelExpressionCheck extends IssuableSubscriptionVisitor {
    * <p>
    * <pre>
    * PropertyPlaceholder ::= Identifier IndexExpression* ("." Identifier IndexExpression*)*
-   * Identifier          ::= [a-zA-Z_][a-zA-Z0-9_]*
+   * Identifier          ::= [a-zA-Z0-9_-]+
```

Comment:
So we can also start with a hyphen `-`? It would be nice to have an example that matches this.